provider/sgx.py

```diff
@@ -0,0 +1,189 @@
+"""
+Module for sgx relevant operations.
+"""
+import logging
+import re
+
+from avocado.utils import process
+from virttest.staging import utils_memory
+from virttest.utils_misc import normalize_data_size
+
+LOG_JOB = logging.getLogger('avocado.test')
```

Comment:
Since class SGXChecker and class SGXTestCapability have parameter "test", I think we can use self.test.log instead. If I was wrong, please correct me. Thanks.

Reply:
I try to replace LOG_JOB with self._test.log.debug("log"), but not work.

Reply:
Yes, I'm debugging it. And I check the vt codes, also using LOG_JOB in class. I think we can keep the same in provider/sgx.py. And in case py file, we can use test.log instead.
I will continue to investigate it. So for the inconvenience.

Reply:
Yes, I also tried self._test.log.debug, it works. And I discussed it with zhenchao, we can use LOG_JOB here. Thanks @zhencliu cc @zixi-chen